src/algebra/group/units_hom.lean

```diff
@@ -41,14 +42,22 @@ variable {M}
 /-- If a map `g : M → units N` agrees with a homomorphism `f : M →* N`, then
 this map is a monoid homomorphism too. -/
 @[to_additive "If a map `g : M → add_units N` agrees with a homomorphism `f : M →+ N`, then this map is an add_monoid homomorphism too."]
-def lift_right (f : M →* N) (g : M → units N) (h : ∀ x, ↑(g x) = f x) :
+def lift_right (f : M →* N) (g : M → units N) (h : ∀ x, f x = g x) :
   M →* units N :=
 { to_fun := g,
-  map_one' := units.ext $ (h 1).symm ▸ f.map_one,
-  map_mul' := λ x y, units.ext $ by simp only [h, coe_mul, f.map_mul] }
+  map_one' := units.ext $ h 1 ▸ f.map_one,
+  map_mul' := λ x y, units.ext $ by simp only [(h _).symm, coe_mul, f.map_mul] }
```

Comment:
It seems that Lean now supports `simp [← h]`.

Reply:
Thanks, I didn't know that!